SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -65,7 +72,7 @@ etc...
     };
```

Comment:
NOTE TO SELF, LAURA - I GOT TO HERE

Reply:
back in the saddle.

Reply:
You're amazing.

Reply:
Thanks, so are you. I didn't get much further than the lines below yesterday. Downed painkillers and back on it now.